srcpkgs/aerc/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'aerc'
+pkgname=aerc
+version=0.1.0
+revision=1
+build_style=go
+go_import_path="git.sr.ht/~sircmpwn/aerc"
+go_ldflags="-X main.Prefix=/usr -X main.ShareDir=/usr/share/aerc"
+hostmakedepends="git scdoc"
+short_desc="Email client for your terminal"
+maintainer="Julio Galvan <juliogalvan@protonmail.com>"
+license="MIT"
+homepage="https://aerc-mail.org/"
+distfiles="https://git.sr.ht/~sircmpwn/aerc/archive/${version}.tar.gz"
+checksum=d33e2e4401c29f3d021e5d0a8aac2a85efa15d72618809278131147d12a0a973
+
+_manpages="aerc.1 aerc-config.5 aerc-imap.5 aerc-smtp.5 aerc-tutorial.7"
+
+post_build() {
+	sed -e 's:@SHAREDIR@:/usr/share/aerc:g' > aerc.conf < config/aerc.conf.in
+	for manpage in $_manpages; do
+		scdoc < doc/${manpage}.scd > $manpage
+	done
+}
+
+post_install() {
+	for manpage in $_manpages; do
+		vman $manpage
+	done
+	vinstall aerc.conf 644 usr/share/aerc
```

Comment:
Why these configuration files are in `/usr/share/aerc` ?

If those are just examples of configuration files, please use `vsconf` instead.

Reply:
I tried that first and everything seemed fine, after installing and running the program i ran into some trouble, those configuration files must be in that path as defined in `go_ldflags` and I verified that reading the source code.